engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -320,28 +323,42 @@ private void initRenderGraph() {
         Node initialPostProcessingNode = nodeFactory.createInstance(InitialPostProcessingNode.class);
         renderGraph.addNode(initialPostProcessingNode, "initialPostProcessingNode");
 
-        // END OF THE SECOND REFACTORING PASS TO SWITCH NODES TO THE NEW ARCHITECTURE - each PR moves this line down.
-        // TODO: node instantiation and node addition to the graph should be handled as above, for easy deactivation of nodes during the debug.
+        Node downSampleSceneAndUpdateExposure = nodeFactory.createInstance(DownSampleSceneAndUpdateExposureNode.class); // TODO: next PR
+        renderGraph.addNode(downSampleSceneAndUpdateExposure, "downSampleSceneAndUpdateExposure");
 
-        Node downSampleSceneAndUpdateExposure = nodeFactory.createInstance(DownSampleSceneAndUpdateExposureNode.class);
         Node toneMappingNode = nodeFactory.createInstance(ToneMappingNode.class);
-        Node bloomPassesNode = nodeFactory.createInstance(BloomPassesNode.class);
-        Node blurPassesNode = nodeFactory.createInstance(BlurPassesNode.class);
-        Node finalPostProcessingNode = nodeFactory.createInstance(FinalPostProcessingNode.class);
-        Node copyToVRFrameBufferNode = nodeFactory.createInstance(CopyImageToHMDNode.class);
-        Node copyImageToScreenNode = nodeFactory.createInstance(CopyImageToScreenNode.class);
-
-        renderGraph.addNode(downSampleSceneAndUpdateExposure, "downSampleSceneAndUpdateExposure");
         renderGraph.addNode(toneMappingNode, "toneMappingNode");
+
+        Node bloomPassesNode = nodeFactory.createInstance(BloomPassesNode.class); // TODO: next PR
         renderGraph.addNode(bloomPassesNode, "bloomPassesNode");
-        renderGraph.addNode(blurPassesNode, "blurPassesNode");
+
```

Comment:
I wish there could've been a better why to initialise this. But it seems like only way to do initialisation. 😔